lib/IDE/Refactoring.cpp

```diff
@@ -2601,6 +2601,46 @@ bool RefactoringActionLocalizeString::performChange() {
   EditConsumer.insertAfter(SM, Target->getEndLoc(), ", comment: \"\")");
   return false;
 }
+    
+bool RefactoringActionDocCommentBoilerplate::isApplicable(ResolvedCursorInfo Tok, DiagnosticEngine &Diag) {
+#warning Implement here
+    if (Tok.Kind != CursorInfoKind::ValueRef)
+        return false;
+    
+    return Tok.ValueD->getKind() == DeclKind::Func;
+}
+    
+bool RefactoringActionDocCommentBoilerplate::performChange() {
+#warning Implement here
+    if(CursorInfo.ValueD->getKind() == DeclKind::Func) {
```

Comment:
Can you invert this condition and return early? We prefer early-returns in LLVM-style codebases.